app.py

```diff
@@ -97,6 +101,36 @@ def transfer_tx_callback(api_keys, tx):
             db.session.add(txnoti)
             db.session.commit()
 
+def get_balance(wallet_address):
+    #TODO
+    #return 0
+    url = '%s/assets/balance/%s/%s' % (app.config["NODE_ADDRESS"], wallet_address, app.config["ASSET_ID"])
+    r = requests.get(url)
```

Comment:
check `r.status_code` for 200, maybe return -1 or something if the request failed